pkg/controller/machine/controller.go

```diff
@@ -65,9 +64,6 @@ func (c *MachineControllerImpl) Init(arguments sharedinformers.ControllerInitArg
 
 	c.linkedNodes = make(map[string]bool)
 
-	// Create machine actuator.
-	// TODO: Assume default namespace for now. Maybe a separate a controller per namespace?
-	c.machineClient = clientset.ClusterV1alpha1().Machines(corev1.NamespaceDefault)
```

Comment:
machineClient remains on the struct, and it looks like it's still used in node.go, I can't see how it's being initialized otherwise, just wanted to check if there is an issue there. (it looks more tricky to remove as well)

Reply:
@dgoodwin , good catch, removed it from struct.
@spew , yes, it's intentional. I want this machine controller can deal with any namespace. 

I will fix & push another commit to fix it

Reply:
@medinatiger still looks like node.go might be using a now uninitialized machineClient field.